### ISSUE TITLE
Seahorse behaves differently on Arch than Manjaro; how to create password keyring with no password?

### ISSUE THREAD

**Issue by BurnyBoi** [dreemurrs-embedded/Pine64-Arch#374]
I have installed Arch+Phosh on my pinephone pro, and I am attempting to create a password keyring that is unlocked by default on login. This is necessary in order for the Fractal app to save/use my login. On Manjaro Phosh, I usually do the following:

1. Install seahorse with `pacman -S seahorse`
2. In Seahorse, create a new password keyring
3. Choose Login for the name
4. Press Continue on the password prompts to confirm I want a blank password on the keyring
5. Plug in a mouse, right click the keyring, click Set As Default

As long as I do these steps, Fractal is able to keep me logged in. However, when I try to do this on Arch, Seahorse doesn't allow me to set a blank password. The Continue button is greyed out until I type a password, unlike on Manjaro which lets me choose blank. Every time I login on Arch, I would now have to start Seahorse, select the keyring, and type a password to unlock it before starting Fractal, which is super annoying.

Basically, from this ticket, I'd like some help creating the default password keyring that is unlocked by default so Fractal works as expected. If there's a way to do this all from the terminal instead I'd be fine with that. Any help would be appreciated.

**Comment by BurnyBoi** [comment]
Figured it out. I just had to type a character, then remove it and it let me pick a blank password for the keyring. Now I can start Fractal without unlocking the keyring first
